omod/src/main/resources/config.xml

```diff
@@ -137,6 +137,13 @@
 			Specifies id of identifier type which will be used for Mpi Person Id
 		</description>
 	</globalProperty>
+	<globalProperty>
```

Comment:
registrationcore is really an api while registrationapp contains web components, the notion of a registration app id really is a UI thing therefore I would assume this GP belongs to the registration app